### PR TITLE
Spotless Linter

### PR DESCRIPTION
**Pull request by ppshobi**
https://github.com/meilisearch/meilisearch-java/issues/19

### CODE REVIEW

**Comment by ppshobi** [comment]
@alallema could you please check and approve the github workflow if its alright?

**Comment by ppshobi** [comment]
@alallema please approve

**Comment by ppshobi** [comment]
@curquiza or @alallema if you could please review the PR, that would be great. 👍

File: src/main/java/com/meilisearch/sdk/Settings.java [hunk 1]
```diff
@@ -1,66 +1,55 @@
...
 import lombok.Setter;
 import org.json.JSONObject;
 
-import java.util.HashMap;
-
 /**
  * Data Structure for the Settings in an {@link Index}
  *
- * Refer https://docs.meilisearch.com/reference/api/settings.html
+ * <p>Refer https://docs.meilisearch.com/reference/api/settings.html
```
Comment (alallema) [inline src/main/java/com/meilisearch/sdk/Settings.java:11]: Why sometimes the Refer is at the end of the line and sometimes it's at the beginning of the next line?

File: src/main/java/com/meilisearch/sdk/SettingsHandler.java [hunk 2]
```diff
@@ -5,66 +5,60 @@
 /**
  * Settings Handler for manipulation of an Index {@link Settings}
  *
- * Refer https://docs.meilisearch.com/reference/api/settings.html
+ * <p>Refer https://docs.meilisearch.com/reference/api/settings.html
```
Comment (alallema) [inline src/main/java/com/meilisearch/sdk/SettingsHandler.java:8]: Like here?

**Review by alallema — COMMENTED** [review]
Thanks, @ppshobi for this PR! So happy to have a linter 😄 
@curquiza is on vacation until the end of the month.

File: src/main/java/com/meilisearch/sdk/Client.java [hunk 3]
```diff
@@ -4,177 +4,174 @@
...
-	 *
-	 * @param uid Unique identifier for the index to create
-	 * @return MeiliSearch API response
-	 * @throws Exception if an error occurs
-	 */
-	public Index createIndex(String uid) throws Exception {
-		return this.createIndex(uid, null);
-	}
+    /**
+     * Creates index Refer https://docs.meilisearch.com/reference/api/indexes.html#create-an-index
```
Comment (alallema) [inline src/main/java/com/meilisearch/sdk/Client.java:29]: I want to say like here sorry

**Review by alallema — APPROVED** [review]

**Review by alallema — APPROVED** [review]

**Comment by alallema** [comment]
bors merge

**Comment by ppshobi** [comment]
Thank you 👍

**Pull request merged**